### DISCUSSION TITLE
Subgraph not indexing correctly

### DISCUSSION THREAD

**Discussion by ligaratus** [f8n/fnd-protocol#188] [Q&A] [answered]
Hi team! I noticed that some of the recent live auctions are not properly indexed in the subgraph, here's a sample query:
```
{
    nfts(
        where: {
          id: "0xd18cf22ef4199fc6201ba900058116c145e33936-8"
        }
    ) {
        id
        bids {
          id
          datePlaced
        }
        mostRecentActiveAuction {
            id
            highestBid {
                amountInETH
            }
            dateStarted
            dateEnding
            auctionId
        }
    } 
}
```
where the result shows that there is an active auction, but there is no bid and null was given for `highestBid`, `dateStarted`, and `dateEnding`.

Here's the piece being queried: https://foundation.app/@withjasmine1/worktime/8

Tested on both [hosted](https://thegraph.com/hosted-service/subgraph/f8n/fnd) and [mainnet playground](https://thegraph.com/hosted-service/subgraph/f8n/f8n-mainnet) graph.

**Comment by HardlyDifficult** [comment, marked as answer]
Issue confirmed, our hosted subgraph has stopped syncing ~22 hours ago. We are checking with The Graph team. Thanks for letting us know!

**Reply by HardlyDifficult** [reply]
The Graph has confirmed this is a problem on their end. They are working to get this back online

**Reply by ligaratus** [reply]
sounds great!

**Reply by HardlyDifficult** [reply]
I forgot to circle back here -- but this should be fully synced and working again. Let us know if you run into any more troubles. Thanks!
